### ISSUE TITLE
Return copy of point rather than reference when not interpolating

### ISSUE THREAD

**Issue by robhawkes** [Turfjs/turf-along#2]
I've been using this module to animate the position of buses along a route and I've noticed a little quirk which has caused me a bit of pain as I couldn't work out the cause. Basically, if the result of `turf.along` requires interpolation then you get returned a new point, otherwise you are returned a reference to a point on the input linestring.

It's caused here: https://github.com/Turfjs/turf-along/blob/master/index.js#L63

The difference is minor, though in my assumption that all returned points were new (not references) I was performing further manipulation on them after running `turf.along` – this ended up changing the referenced points in the original linestring and causing unexpected issues.

In short, it would be good to either make it obvious in the readme that the returned point may be a direct reference (so don't manipulate it) or otherwise update the code to return a copy of the non-interpolated point instead.

I don't mind submitting a PR, though I'd first like to know how you want to approach this 😊

**Comment by tmcw** [comment]
Fixed in https://github.com/Turfjs/turf/commit/948402daeb288db52b118bbc0c047fda35c14331
